File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        
        i=0
        j=len(numbers)-1
        
        while i<j:
            su=numbers[i]+numbers[j]
            
            if su==target:
                return [i+1,j+1]
            if su>target:
                j-=1
            else:
                i+=1
        return []
        

File: 167-two-sum-ii-input-array-is-sorted/167-two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
    int startInc=0,endDec=numbers.length-1;
    while (startInc<endDec) {
      if(numbers[startInc]+numbers[endDec]==target){
        return new int[]{startInc+1,endDec+1};
      } else if (numbers[startInc]+numbers[endDec]<target) {
        startInc++;
      } else if (numbers[startInc]+numbers[endDec]>target) {
        endDec--;
      }
    }
    return new int[]{-1,-1};  
    }
}